sb-cb-ui-consumption/library/sunbird-cb/consumption/src/lib/_common/cards/card-progress-portrait-lib/card-progress-portrait-lib.component.html

```diff
@@ -1,114 +1,120 @@
-                <mat-icon class="mat-icon main_icon mr-1">access_time</mat-icon>
-                <span class=" ws-mat-white-text duration">
-                  {{widgetData?.content?.programDuration}} {{widgetData?.content?.programDuration === 1 ? 'day' : 'days'}}
-                </span>
-              </div>
-              <!-- <ng-container *ngIf="cbPlanMapData && cbPlanMapData[widgetData?.content?.identifier]">
-                    <ng-container [ngTemplateOutlet]="cbpPlan" [ngTemplateOutletContext]="{data: {
-                      cbpData: cbPlanMapData[widgetData?.content?.identifier]
-                    }}"></ng-container>
-              </ng-container> -->
-            </div>
-          </ng-container>
-          <ng-container *ngIf="!widgetData.content.posterImage">
-            <ng-container *ngIf="widgetData.content.appIcon; else defaultImg">
-              <img mat-card-image [src]="widgetData.content.appIcon | pipePublicURL" loading="lazy"
-              class="card-image ws-mat-primary-lite-background margin-remove"  [wsUtilsDefaultThumbnail]="defaultThumbnail"
-              [alt]="widgetData.content.name" />
-            </ng-container>
-            <ng-template #defaultImg>
-              <img mat-card-image [src]="defaultThumbnail" loading="lazy"
-              class="card-image ws-mat-primary-lite-background margin-remove"
-              [alt]="widgetData.content.name" />
-            </ng-template>
-          </ng-container>
+      <div class="flex w-full  flex-middle">
+        <div class="portrait_course_logo_box margin-right-xs">
+          <img [src]="widgetData?.content?.creatorLogo | pipePublicURL " class="source-icon"  loading="lazy" [wsUtilsDefaultThumbnail]="defaultSLogo"
+          [alt]="(widgetData?.content?.sourceName + '_' + widgetData?.content?.identifier)" />
         </div>
-        <div class="width-1-1">
-          <div [id]="'m-c-'+ widgetData.content?.identifier"
-              class="portrait-subheading mat-subheading-2 title-text">
-              {{ widgetData.content.name }}
-          </div>
-          <div class="flex w-full  flex-middle">
-            <div class="portrait_course_logo_box margin-right-xs">
-              <img [src]="widgetData.content.creatorLogo | pipePublicURL " class="source-icon"  loading="lazy" [wsUtilsDefaultThumbnail]="defaultSLogo"
-              [alt]="(widgetData.content.sourceName + '_' + widgetData.content.identifier)" />
-            </div>
-            <span class="portrait-org-name font-normal mat-caption">{{'cardcontentv2.by' | translate}} {{ (widgetData.content.organisation && widgetData.content.organisation[0]) ? widgetData.content.organisation[0] : 'Karmayogi Bharat' }}</span>
-          </div>
-
+        <span class="portrait-org-name font-normal mat-caption">{{'cardcontentv2.by' | translate}} {{ (widgetData?.content?.organisation && widgetData?.content?.organisation[0]) ? widgetData?.content?.organisation[0] : 'Karmayogi Bharat' }}</span>
```

Comment:
'Karmayogi Bharat' should come from configuration for library, there should be any hardcoded text